sos/plugins/tomcat.py

```diff
@@ -34,6 +38,11 @@ def setup(self):
         log_glob = "/var/log/tomcat*/catalina.out"
         self.add_copy_spec_limit(log_glob, sizelimit=limit)
 
+        if self.get_option("log"):
+            self.add_copy_spec([
```

Comment:
Couple of comments - an `add_copy_spec()` call for a single item doesn't need the item to be wrapped in a list (the API accepts either a single string or a list of strings), so just:

```python
        self.add_copy_spec("/var/log/tomcat*/*")
```

is fine.

More generally though I think this patch needs more work - if the catalina logs are now obsolete there is really no point in collecting them at all (at least on newer installations) so we would still end up collecting nothing if the user did not specify `--all-logs`: I think there needs to be some middle ground between effectively collecting nothing and collecting absolutely everything (via globs and with all size limits disabled).